.github/workflows/release.yml

```
@@ -203,10 +180,12 @@ jobs:
           echo "ninja:" && ninja --version
           echo "clang:" && clang --version
 
-      - name: download llvm
-        run: |
-          curl -L -o llvm.tar.gz "${{ steps.get-llvm-musl-artifact.outputs.result }}"
-          tar -xvf llvm.tar.gz
+      - name: get llvm gnu
```

```suggestion
      - name: get llvm musl
```

I'm seeing musl below